satpy/composites/__init__.py

```diff
@@ -647,6 +659,17 @@ def _get_reflectance(self, projectables, optional_datasets):
 class NIREmissivePartFromReflectance(NIRReflectance):
     """Get the emissive par of NIR bands."""
 
+    def __init__(self, **kwargs):
```

Comment:
You can make max_sza explicit:
```suggestion
    def __init__(self, max_sza=None, **kwargs):
```